tests/default_vars.sh

```diff
@@ -1028,7 +1143,7 @@ export atm_model=datm
 export CPLMODE=ufs.nfrac.aoflux
 
 # datm defaults
-export INPUT_NML=input.mom6.nml.IN
+export INPUT_NML=cpld_control.nml.IN
```

Comment:
Do we still have this file? I think cpld_control.nml.IN is changed to global_control.nml.IN?

Reply:
@junwang-noaa I wondered about the mom6 input nml for the DATM also, but since Dan didn't report those tests as needing a new baseline, I assumed that somehow this was working for DATM.

Reply:
I'll just go ahead and revert the change back to input.mom6.nml.IN and then remove the cpld_control.nml.IN file. 

That way we remove the file and keep the scope of the PR changes to the fv3 global runs. We can worry about the datm tests in another PR if/when we consolidate those.

Reply:
> I kept the file for now because I didn't want to do any extensive changes to the datm cases in the PR. I can test the datm runs using the global_control.nml.IN and get rid of the cpld_control.nml.IN file when those cases are working with the new global namelist file.

I'm a little confused...the datm tests will never use the global namelist file, will they?